### DISCUSSION TITLE
Search Engine Choice - Bing

### DISCUSSION THREAD

**Discussion by Antonimo-Dev** [zen-browser/desktop#1616] [Q&A] [answered]
Hi,

How can I choose to use Bing as the default search engine? It does not appear in the Settings » Search options.

Thanks,

**Comment by QW4D** [comment, marked as answer]
at the end of the search settings page there is a list of search services, click the "add" button and add bing there

**Reply by Antonimo-Dev** [reply]
Thank you!

**Reply by geekonloose** [reply]
Hi, could you guide me, what should I enter in Engine URL?

**Reply by QW4D** [reply]
> Hi, could you guide me, what should I enter in Engine URL?

https://www.bing.com/search?q=%s
or, alternatively, you can install  [this extension](https://addons.mozilla.org/en-US/firefox/addon/bing-search-engine-/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)

**Reply by geekonloose** [reply]
Thank you..!!
